### ISSUE TITLE
apiserver crashed when priority classs already exists

### ISSUE THREAD

**Issue by hchenxa** [kubernetes/kubernetes#67367]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
/kind bug
> /kind feature

**What happened**:
When I start the apiserver, found apiserver crased due to the priority class already exists.
```
E0814 06:00:23.516116       1 autoregister_controller.go:190] v1.autoscaling failed with : apiservices.apiregistration.k8s.io "v1.autoscaling" already exists
E0814 06:00:23.516317       1 autoregister_controller.go:190] v1beta1.batch failed with : apiservices.apiregistration.k8s.io "v1beta1.batch" already exists
E0814 06:00:23.516443       1 autoregister_controller.go:190] v2alpha1.batch failed with : apiservices.apiregistration.k8s.io "v2alpha1.batch" already exists
F0814 06:00:23.517195       1 hooks.go:188] PostStartHook "scheduling/bootstrap-system-priority-classes" failed: unable to add default system priority classes: priorityclasses.scheduling.k8s.io "system-cluster-critical" already exists
```

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`): 1.11.1
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release): ubuntu 16.04
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by hchenxa** [comment]
sig api-machinery

**Comment by yue9944882** [comment]
hi @hchenxa, do you happen to have multiple apiservers start at the same time?

**Comment by hchenxa** [comment]
yes, I have a HA environment which have 3 apiserver.

**Comment by yue9944882** [comment]
sadly, it's the racing then. might launch your servers one-by-one to avoid it for now.

**Comment by liggitt** [comment]
Yes, and should be picked to release branches once complete

**Comment by tanshanshan** [comment]
fixed it in #67372 as @Huang-Wei  said.
Thanks
